tests/kuttl/tests/keystone_scale/04-scale-down-keystoneapi.yaml

```diff
@@ -0,0 +1,5 @@
+apiVersion: kuttl.dev/v1beta1
+kind: TestStep
+commands:
+  - script: |
+      oc patch keystoneapi -n openstack keystone --type='json' -p='[{"op": "replace", "path": "/spec/replicas", "value":1}]'
```

Comment:
Could you add a separate step where you assert that scaling down to 0 replicas works as well?

Reply:
Done!

Reply:
Awesome. Thanks!